tests/Get-DbaDbSpace.Tests.ps1

```diff
@@ -24,30 +24,30 @@ Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
     }
     #Skipping these tests as internals of Get-DbaDbSpace seems to be unreliable in CI
     Context "Gets DbSpace" {
-        $results = Get-DbaDbSpace -SqlInstance $script:instance2 | Where-Object {$_.Database -eq "$dbname"}
-        It -skip "Gets results" {
+        $results = Get-DbaDbSpace -SqlInstance $script:instance2 | Where-Object { $_.Database -eq "$dbname" }
+        It "Gets results" {
             $results | Should Not Be $null
         }
         foreach ($row in $results) {
-            It -skip "Should retreive space for $dbname" {
+            It "Should retreive space for $dbname" {
                 $row.Database | Should Be $dbname
             }
-            It -skip "Should have a physical path for $dbname" {
+            It "Should have a physical path for $dbname" {
                 $row.physicalname | Should Not Be $null
             }
         }
     }
     #Skipping these tests as internals of Get-DbaDbSpace seems to be unreliable in CI
     Context "Gets DbSpace when using -Database" {
         $results = Get-DbaDbSpace -SqlInstance $script:instance2 -Database $dbname
-        It -skip "Gets results" {
+        It "Gets results" {
             $results | Should Not Be $null
         }
         Foreach ($row in $results) {
-            It -skip "Should retreive space for $dbname" {
+            It "Should retreive space for $dbname" {
                 $row.Database | Should Be $dbname
```

Comment:
maybe we should check if "space" is retrieved as well ? not a fixed value, of course, just "not null"